pkg/v1/cluster/testing/fixtures.go

```diff
@@ -32,7 +32,15 @@ const testGetClusterResponseRaw = `
         "enable_patch_version_auto_upgrade": true,
         "zonal": false,
         "kubernetes_options": {
-            "enable_pod_security_policy": true
+            "enable_pod_security_policy": true,
+			"feature_gates": [
```

Comment:
Let's use spaces for indentation here and in other similar pieces below. Tab characters break format of these JSON fixtures a little. You can view it more clearly using `git log -1 -p` command.